src/main/webapp/portal-core/js/portal/layer/querier/QueryTargetHandler.js

```diff
@@ -120,17 +120,53 @@ Ext.define('portal.layer.querier.QueryTargetHandler', {
                 continue;
             }
 
-
             point = Ext.create('portal.map.Point', {
                 latitude : queryTargets[i].get('lat'),
                 longitude : queryTargets[i].get('lng')
             });
 
+
             var shortTitle = cswRecord.get('name');
+            
+            // abbreviate the service provider name
+            var provider = cswRecord.get('contactOrg');   
+            
+            // list of service providers 
+            var SERVICE_PROVIDERS = [
+                'CSIRO',
+                'Geoscience Australia',
+                'Northern Territory',
+                'Queensland',
+                'South Australia',
+                'Tasmania',
+                'Victoria',
+                'Western Australia'
+            ];
+            
+            for (var i = 0; i < SERVICE_PROVIDERS.length; i++) {                
```

Comment:
Or better still (from http://ryanmorr.com/true-hash-maps-in-javascript/ - we are using ES5.  ES6 has improvements as noted at the bottom).  Its better because the providers object is bare and is not pre-loaded with all the methods and attributes from the prototype chain:

    var providers=Object.create(null);
    providers["NSW"]="New South Wales";
    ....
    if (value in providers) {
          provider = providers[value];
    }